starfish/core/experiment/builder/__init__.py

```diff
@@ -28,13 +35,18 @@
 from .providers import FetchedTile, TileFetcher
 
 
-AUX_IMAGE_NAMES = {
-    'nuclei',
-    'dots',
-}
 DEFAULT_DIMENSION_ORDER = (Axes.ZPLANE, Axes.ROUND, Axes.CH)
 
 
+@dataclass
+class TileCoordinate:
```

Comment:
I think using the word Coordinates here is a little confusing since we already have a concept of tile coordinate...maybe `TileIndex` or `TileIndentifier`?

Reply:
I like `TileIdentifier`.